### ISSUE TITLE
No predictions

### ISSUE THREAD

**Issue by kcambrek** [Babelscape/rebel#35]
Hi, I am recreating what you did, but in Dutch.  The training script is running, but the model does not predict a thing, even after several epochs. Additionally, the loss does not change at all.

![image](https://user-images.githubusercontent.com/50198133/184494635-faa145ff-ebbf-4f6d-8924-cdc0652895e9.png)

What I changed to the config files is the paths to my data and the name of the models (from bart-large to mbart-50). Did I miss anything? The data is generated by Crocodile and has the correct structure.

I hope you have some time to take a look.

**Comment by LittlePea13** [comment]
The fact the loss doesn't change at all seems concerning. The overall approach should work as well for mbart, so there has to be some issue with training. I suggest you debug the loss, see if it returns any NaN. This may be caused by some mishandling of special tokens. Learning rate may also need to be changed. Also remember that mbart expects the language token at the start of the input/output. 

But again, if loss is not changing, I would focus on that rather than checking the predictions.

**Comment by LittlePea13** [comment]
It seems this has become stale, hence I am closing the issue. Feel free to share what the issue was if you found it or reopen it in case you have more details.
